apps/site/assets/ts/schedule/schedule-loader.tsx

```diff
@@ -100,6 +101,7 @@ const renderDirectionAndMap = (
       ratingEndDate={ratingEndDate}
       stops={stops}
       today={today}
+      scheduleNote={scheduleNote!}
```

Comment:
Why is the not-null override needed here? Are we guaranteed to always have a scheduleNote at this point?